src/main/drivers/camera_control.c

```diff
@@ -146,8 +147,16 @@ void cameraControlInit(void)
     }
 }
 
-void cameraControlProcess(uint32_t currentTimeUs)
+void cameraControlProcess(timeUs_t currentTimeUs)
 {
+
+#if defined(USE_RCDEVICE)
```

Comment:
Should go in a different "task". Not really related to camera control code, user may run both an FPV camera and RC Split for some weird reason and we shouldn't disable usual camera control in this case.